templates/kubernetes/terraform/environments/stage/main.tf

```diff
@@ -32,4 +32,19 @@ module "kubernetes" {
 
   # Logging configuration
   logging_type = "<% index .Params `loggingType` %>"
+
+  # Application policy list
+  application_policy_list = [
+    {
+      service_account = "app1"
```

Comment:
How about we change this service account name to "backend-service" or something and then also change the zero-deployable-backend project to use this service account?